atr/get/distribution.py

```diff
@@ -94,27 +86,80 @@ async def list_get(session: web.Committer, project: str, version: str) -> str:
             shared.distribution.html_tr_a("Web URL", dist.web_url),
         ]
         block.table(".table.table-striped.table-bordered")[tbody]
-        form_action = util.as_url(post.distribution.delete, project=project, version=version)
-        delete_form_element = forms.render_simple(
-            delete_form,
-            action=form_action,
-            submit_classes="btn-danger",
-        )
-        block.append(htm.div(".mb-3")[delete_form_element])
+
+        # TODO: add a confirm: bool keyword argument to form.render, allowing us to
```

Comment:
This should be fairly trivial, so perhaps it would be a better idea to implement that within this PR? Otherwise there's a big block of code that we're adding here which will have to be entirely changed.